src/utilities/permissions.ts

```diff
@@ -533,6 +574,13 @@ const queryPermissions: Record<GQLKeys, (user: User | null, ...args: any[]) => b
         (isPlanOwner(user, plan) || isPlanCollaborator(user, plan)))
     );
   },
+  DELETE_PLAN_DERIVATION_GROUP: (user: User | null, plan: PlanWithOwners): boolean => {
+    return (
+      isUserAdmin(user) ||
+      (getPermission([Queries.DELETE_PLAN_DERIVATION_GROUP], user) &&
```

Comment:
Oh god sorry, I think I was looking at the wrong place that this query was being used. I think what actually needs to happen is for the `DELETE_PLAN` permission, it needs `Queries.DELETE_PLAN_DERIVATION_GROUP` to be included since that was added to the `DELETE_PLAN` query.